sale_stock_prebook/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "sale_stock_prebook",
```

Comment:
Following the common nomenclature.
```suggestion
    "name": "Sale Stock Prebook",
```
Or "Prebooking Stock for Draft Sales Orders" or something?